src/renderer/services/midgard/actions.ts

```diff
@@ -0,0 +1,62 @@
+import * as RD from '@devexperts/remote-data-ts'
```

Comment:
NOte: Almost no change in this file. This file has just been renamed `poolActionsHistory` -> `actions`. Also it's state handling is in `useMidgardHistory` hook.